lua/nvim-tree/git/runner.lua

```diff
@@ -62,19 +73,25 @@ function Runner:_getopts(stdout_handle, stderr_handle)
   }
 end
 
+---@param output string
 function Runner:_log_raw_output(output)
   if log.enabled "git" and output and type(output) == "string" then
     log.raw("git", "%s", output)
     log.line("git", "done")
   end
 end
 
+---@param callback function|nil
 function Runner:_run_git_job(callback)
   local handle, pid
   local stdout = vim.loop.new_pipe(false)
   local stderr = vim.loop.new_pipe(false)
   local timer = vim.loop.new_timer()
 
+  if stdout == nil or stderr == nil or timer == nil then
+    return
+  end
```

Comment:
Added this little check to silence the "Need nil check" warnings.

Reply:
This is where we really gain the value.